xformers/components/nvfuser/bias_dropout_res.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import logging
+from typing import Any, Optional
+
+import torch
+import torch.nn as nn
+from functorch.compile import memory_efficient_fusion
+
+from xformers.components import LayerNormStyle
+
+
+def _fn(
```

Comment:
Perhaps we can pull out a general method that takes in a function and outputs the nvfused version. Then we can use that to provide these various fusions

Reply:
It could remove some redundancy in these files